frappe/custom/doctype/customize_form/customize_form.js

```diff
@@ -87,23 +79,16 @@ frappe.ui.form.on("Customize Form", {
 		frm.trigger("setup_default_views");
 	},
 
-	setup_sortable: function (frm) {
+	add_customize_child_table_button: function (frm) {
```

Comment:
Note to self. This needs to be supported in form builder too. 

@shariquerik

Reply:
added it as a todo.